README.md

```diff
@@ -467,57 +299,86 @@ pg_dump:
 PGHOST
 PGPORT
 PGUSER
+PGDATABASE
 ```
 
 It is also possible to configure access via pg_dump's command-line options
 `--host`, `--port`, and `--username`.
 
-### 2. Verify pg_dump output
+#### 1.2 Direct access to PostgreSQL
+
+In this case, HarbourBridge connects directly to the PostgreSQL
+database to retrieve table schema and data using the setting of
+standard _PGHOST_, _PGPORT_, _PGUSER_, _PGDATABASE_ environment
+variables. Password can be specified either in the _PGPASSWORD_
+environment variable or provided at the password prompt.
+
+#### 1.3 mysqldump
+
+If you are using mysqldump (-driver=mysqldump), check that mysqldump is
+correctly configured to connect to your MySQL
+database. Note that mysqldump uses the same options as mysql to connect to your
+database. See the [mysql](https://dev.mysql.com/doc/refman/8.0/en/mysql-commands.html) and
+[mysqldump](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html) documentation.
+
+Access to a MySQL database is configured via mysqldump's command-line options `--host`,
```

Comment:
Maybe merge this sentence in with the previous paragraph? It seems kind of an afterthought here?

How about:

If you are using mysqldump (-driver=mysqldump), check that mysqldump is
correctly configured to connect to your MySQL database via the
command-line options `--host`, `--port`, and `--user`.
Note that mysqldump uses the same options as mysql to connect to your
database. See the [mysql](https://dev.mysql.com/doc/refman/8.0/en/mysql-commands.html) and
[mysqldump](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html) documentation.

Reply:
Yup, looks good. Done.